### ISSUE TITLE
Support versions > 232.x

### ISSUE THREAD

**Issue by Pokom** [zzehring/intellij-jsonnet#71]
I've updated GoLand to 2023.3 and see the following error: 

```
Plugin 'Jsonnet Language Server' (version '0.1.5') is not compatible with the current version of the IDE, because it requires build 232.* or older but the current build is GO-233.11799.228
```

We should probably add support for the latest versions 😄

**Comment by Pokom** [comment]
Looks like @charleskorn implemented this in #70. @zzehring any chance we can get that merged in and released?

**Comment by zzehring** [comment]
Yep! This was done in https://github.com/zzehring/intellij-jsonnet/pull/73 and available in v0.1.6.
